solutions/devpsrint-michael-douglas-1/GitHubAppTests/SnapshotTests/LoadingViewTest.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoadingViewTest.swift
+//  GitHubAppTests
+//
+//  Created by Fabio Martinez on 26/01/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import GitHubApp
+
+class LoadingViewTest: XCTestCase {
+
+    let snapshotSize = CGSize(width: 500, height: 200)
```

Comment:
Lembra que expliquei sobre o `sut`? Então, crie uma propriedade `sut` e dai dentro do seu método tu só chama ela pra fazer os asserts.

```suggestion
    lazy var sut: LoadingView = {
       let view = LoadingView(width: 500, height: 200)
       return view
    }()
```

Reply:
Feita a implementação do sut